jazon-core/src/main/java/com/zendesk/jazon/expectation/Expectations.java

```diff
@@ -0,0 +1,8 @@
+package com.zendesk.jazon.expectation;
+
+public class Expectations {
+    
+    public static AnyNumberOf anyNumberOf(Object element) {
+        return new AnyNumberOf(element);
```

Comment:
The reason why `ArrayEachElementExpectation` cannot be returned directly is that `ExpactationFactory` is not available at this point

Reply:
Oh that is true, we cannot construct the expectation-instance out of static context. My bad to suggest that :)